pkg/scheduler/api/helpers/helpers.go

```diff
@@ -49,6 +49,7 @@ func Max(l, r *api.Resource) *api.Resource {
 
 	res.MilliCPU = math.Max(l.MilliCPU, r.MilliCPU)
 	res.Memory = math.Max(l.Memory, r.Memory)
+	res.EphemeralStorage = math.Max(l.EphemeralStorage, r.EphemeralStorage)
```

Comment:
Please update other resource comparsion functions as well such as `Min` `Less` and so on.